.github/workflows/create-release.yml

```diff
@@ -0,0 +1,38 @@
+name: Create Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'The new version to tag, ex: 1.0.5'
+        required: true
+        type: string
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Create Release
+        run: |
+          set -euo pipefail
+
+          git config user.name "Automatic version bump"
+          git config user.email "keithbsmiley@gmail.com"
```

Comment:
How about using a generic email like noreply@github.com since this is an automated commit?